src/components/landing/FeatureBlocks/AlternatingFeatureBlock.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Lead, DotPattern, Title } from 'components';
+import classnames from 'classnames';
+
+import Bullet from './FeatureBullet';
+
+const FeatureHeader = ({ title, description }) => (
+  <>
+    <Title el="h3">{title}</Title>
+
+    <div className="mt-3">
+      <Lead size="small">
+        {description}
+      </Lead>
+    </div>
+  </>
+);
+
+const FeatureBlock = ({
+  title,
+  description,
+  bullets,
+  illustration,
+  illustrationSide = 'right',
+}) => (
+  <div className={classnames('lg:grid lg:grid-cols-2 lg:gap-8 lg:items-center', { 'lg:grid-flow-row-dense': illustrationSide === 'left' })}>
+    <div className={classnames({ 'lg:col-start-2': illustrationSide === 'left', 'relative': illustrationSide === 'right' })}>
+      <FeatureHeader
+        title={title}
+        description={description}
+      />
+
+      <dl className="mt-10 space-y-10">
+        {bullets.map((item) => (<Bullet item={item} key={item.name} />))}
+      </dl>
+    </div>
+
+    <div
+      className={
+        classnames('mt-10 -mx-4 relative lg:mt-0', { 'lg:col-start-1': illustrationSide === 'left' })
+      }
+      aria-hidden={illustrationSide === 'right'}
+    >
+      <DotPattern
+        className="absolute left-1/2 transform -translate-x-1/2 translate-y-16 lg:hidden"
+        width={784}
+        height={404}
+        aria-hidden={illustrationSide === 'left'}
+        id="ca9667ae-9f92-4be7-abcb-9e3d727f2941"
```

Comment:
This will possibly introduce multiples of the same ids to the site if there are many of these components used. Better to derive it from something instead. (Or drop it if it's not used anywhere). There are few more of these below also

Reply:
I'm gonna skip over this comment since I'm just moving this code rather than adding this.

But you are right that something is not good here.